cristin-person/src/main/java/no/unit/nva/cristin/person/model/nva/Affiliation.java

```diff
@@ -0,0 +1,108 @@
+package no.unit.nva.cristin.person.model.nva;
+
+import static nva.commons.core.attempt.Try.attempt;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import java.net.URI;
+import java.util.Objects;
+import no.unit.nva.cristin.person.model.cristin.CristinAffiliation;
+import nva.commons.core.JacocoGenerated;
+
+@JacocoGenerated
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class Affiliation {
+
+    private final URI organization;
+    private final Boolean active;
+    private final Role role;
+
+    /**
+     * Creates an Affiliation for serialization to client.
+     *
+     * @param organization Identifier of Organization.
+     * @param active       If this affiliation is currently active.
+     * @param role         What roles this Person has at this affiliation.
+     */
+    @JsonCreator
+    public Affiliation(@JsonProperty("organization") URI organization, @JsonProperty("active") Boolean active,
+                       @JsonProperty("role") Role role) {
+        this.organization = organization;
+        this.active = active;
+        this.role = role;
+    }
+
+    public URI getOrganization() {
+        return organization;
+    }
+
+    public Boolean getActive() {
+        return active;
+    }
+
+    public Role getRole() {
+        return role;
+    }
+
+    @JacocoGenerated
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (!(o instanceof Affiliation)) {
+            return false;
+        }
+        Affiliation that = (Affiliation) o;
+        return Objects.equals(getOrganization(), that.getOrganization())
+            && Objects.equals(getActive(), that.getActive())
+            && Objects.equals(getRole(), that.getRole());
+    }
+
+    @JacocoGenerated
+    @Override
+    public int hashCode() {
+        return Objects.hash(getOrganization(), getActive(), getRole());
+    }
+
+    @JacocoGenerated
+    public static final class Builder {
```

Comment:
Is Builder necessary for an object of 3 values? In any case a it is better to have Builders that mutate the target object using the object's setters. So each withX() calls setX() and build() just returns the object.